packages/epics/src/treasury/components/list-requests.tsx

```diff
@@ -0,0 +1,122 @@
+'use client';
+import { useState } from 'react';
+import { Text } from '@radix-ui/themes';
+import { PlusIcon } from '@radix-ui/react-icons';
+import { Button, FilterMenu } from '@hypha-platform/ui';
+import { CardRequest } from './card-request';
+
+type ListRequestsProps = Record<string, never>;
+
+type OptionType = {
+  label: string,
+  value: string
+}
+
+type FilterType = {
+  value: string,
+  options: OptionType[]
+}
+
+const requestsfilterSettings: FilterType = {
+  value: 'most-recent',
+  options: [
+    { label: 'All', value: 'all' },
+    { label: 'Most recent', value: 'most-recent' }
+  ],
+};
+
+const dummyData = {
+  totalValue: 15850,
+  requests: [
+    {
+      avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+      name: 'Name',
+      surname: 'Surname',
+      value: 1950,
+      symbol: 'ETH',
+      date: '2024/09/23'
+    },
+    {
+      avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+      name: 'Name',
+      surname: 'Surname',
+      value: 1950,
+      symbol: 'ETH',
+      date: '2024/09/23'
+    },
+    {
+      avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+      name: 'Name',
+      surname: 'Surname',
+      value: 1950,
+      symbol: 'ETH',
+      date: '2024/09/23'
+    },
+    {
+      avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+      name: 'Name',
+      surname: 'Surname',
+      value: 1950,
+      symbol: 'ETH',
+      date: '2024/09/23'
+    },
+  ]
+}
+
+export const ListRequests: React.FC<ListRequestsProps> = () => {
+  const [requests, setRequests] = useState(dummyData.requests);
+  const loadMoreRequests = () => {
+    const newRequests = [
+      {
+        avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+        name: 'Name',
+        surname: 'Surname',
+        value: 1950,
+        symbol: 'BTC',
+        date: '2024/09/23'
+      },
+      {
+        avatar: 'https://images.unsplash.com/photo-1544005313-94ddf0286df2?&w=64&h=64&dpr=2&q=70&crop=faces&fit=crop',
+        name: 'Name',
+        surname: 'Surname',
+        value: 1950,
+        symbol: 'BTC',
+        date: '2024/09/23'
+      },
+    ];
+    setRequests(prevRequests => [...prevRequests, ...newRequests]);
+  }
```

Comment:
> It looks like we're introducing a bad pattern here. The logic for loading more data should not be concern of this display component. The component should only receive data, and delegate users desire to load more, and then again receive the data.

This is a good point, but since I'm working with mock data I had to put it here to avoid making the parent component client-side. The general idea was that List-prefixed components would work with the data of the entity they are responsible for, but I'm not against any other solution if you suggest :)